### ISSUE TITLE
Doesn't work since a few days. 

### ISSUE THREAD

**Issue by jean-baptiste-d** [Processus-Thief/vulnspy#6]
Hello, 

I have a little problem with the script. It doesn't work since a few days. 

![image](https://user-images.githubusercontent.com/34312282/212475644-dc47fa66-4353-4b33-965d-15a29915ea38.png)

After some investigation, I think that the function: "scrape_cve". Doesn't work properly anymore. Because it returns an empty array. 

![image](https://user-images.githubusercontent.com/34312282/212475804-2f74ee90-91a9-42c4-9814-3fd98839df32.png)

If I find a solution, I'll put it here. Thank you.

**Comment by ProcessusT** [comment]
Yep, the CERT-FR change its links so the scraping has to be updated 😉 but for the moment I didn't have time to do it
